frontend/ui/widget/screensaverwidget.lua

```diff
@@ -8,6 +8,7 @@ local UIManager = require("ui/uimanager")
 local Screen = Device.screen
 
 local ScreenSaverWidget = InputContainer:new{
+    name = "Screensaver",
```

Comment:
```suggestion
    name = "ScreenSaver",
```

(Matches the class name)